starter-api/src/main/resources/META-INF/native-image/io.micronaut.starter.api/micronaut-starter-api/reflect-config.json

```diff
@@ -43,10 +43,5 @@
 },{
 "name" : "org.apache.commons.compress.archivers.zip.ResourceAlignmentExtraField",
 "allDeclaredConstructors" : true
-},{
-"name":"io.micronaut.starter.api.preview.PreviewDTO",
```

Comment:
is this no longer necessary?

Reply:
It is not necessary anymore. That was added because of this issue: https://github.com/micronaut-projects/micronaut-starter/pull/830. Since I cannot reproduce the issue anymore, I removed the metadata. I also created a test which confirms everything works ok: https://github.com/micronaut-projects/micronaut-starter/blob/bc2bec28cc605a2878a04a9cc0989c3e5e48c1a1/test-suite-graal/src/test/java/example/StarterWebNettyTest.java#L65